### ISSUE TITLE
Request: Create a Flatpak for automatic updates and easier install

### ISSUE THREAD

**Issue by Dyras** [c0re100/qBittorrent-Enhanced-Edition#285]
Hello. I think it would be great if qBittorrent EE could be installed as a Flatpak. Automatic updates would be neat, for example.

**Comment by abcfy2** [comment]
AppImage already support zsync. You can use [AppimageUpdate](https://github.com/AppImage/AppImageUpdate/releases) or others could support this feature.

**Comment by Dyras** [comment]
Indeed, but Fedora Silverblue for example doesn't support AppImage in the same way that it supports Flatpak. Just think it would be a fun addition, since it's slowly becoming the norm.

**Comment by hyknn** [comment]
Anyone making this?
